sections/04_study.md

```diff
@@ -294,11 +294,65 @@ unsupervised uses.`
 
 ### Single-cell
 
-*There are not many neural network papers in this area (yet), unless we count
-imaging applications.  But there is still plenty to discuss.  The existing
-methods [@tag:Arvaniti2016_rare_subsets @tag:Angermueller2016_single_methyl]
-use interesting network architectures to approach single-cell data.
-[@tag:Shaham2016_batch_effects] could fit here.*
+Single-cell methods are generating extreme excitement as scientists recognize
+the vast heterogeneity within unicellular species and between cells of the same
+tissue type in the same organism [@tag:Gawad2016_singlecell]. For instance,
+tumors harbor extensive heterogeneity that can give rise to metastatic
+subclones, and adjacent neurons in the brain can contain different somatic
+variants due to random mutation [@tag:Lodato2015_neurons]. Understanding
+single-cell diversity in all its dimensions — genetic, epigenetic,
+transcriptomic, proteomic, morphologic, and metabolic — is key if precision
+medicine is to be targeted not only to a specific individual, but also to
+specific pathological subsets of cells.
+
+Single-cell methods also promise to uncover a wealth of new biological
+knowledge. Temporality is often difficult to study in biology due to the speed
+at which biological reactions occur. Luckily, a sufficiently large population of
+single cells will have enough representative “snapshots” to recreate timelines
+of biological processes such as mitosis. If tracking processes over time is not
+the limiting factor, single cell techniques can provide maximal resolution
+compared to those that average across all cells in bulk tissue: this enables the
+study of transcriptional bursting with single-cell FISH or the heterogeneity of
+epigenetic patterns with single-cell Hi-C, ATAC-seq, or ChIP-seq
+[@tag:Liu2016_sc_transcriptome @tag:Vera2016_sc_analysis]. Finally, examining
+populations of single cells can reveal biologically meaningful subsets of cells
+as well as their underlying gene regulatory networks
+[@tag:Arvaniti2016_rare_subsets @tag:Gaublomme2015_th17].
```

Comment:
Whether now or in a TODO, I think we should say more about what the neural network contributes here.  Otherwise a lot of the section is an overview of single cell biology and technologies but not directly related to deep learning.  There was a lot of discussion in #79 to use as a starting point.